core/src/opcode.cpp

```diff
@@ -254,6 +256,7 @@ MemoryAccess get_memory_access(const Family family) {
     case Family::DEY:
     case Family::DEX:
     case Family::EOR:
+    case Family::ROL:
```

Comment:
It doesn't really matter for implied/accumulator that you implemented in this PR, but ROL is a read/write instruction so it should return ReadWrite here.

Reply:
Ok, thanks! Will fix!